api/lib/database/methods.js

```diff
@@ -389,6 +389,44 @@ const getUser = ({ hasura_url, hasura_admin_key }) => (schema) => async (
   )
 }
 
+// write and return a new session
+const getNewSession = ({ hasura_url, hasura_admin_key }) => () => async (
```

Comment:
```suggestion
const storeNewSession = ({ hasura_url, hasura_admin_key }) => () => async (
```
It is actually storing the new session, isn't it?

Reply:
oops I see